src/llrt-function.ts

```diff
@@ -154,6 +163,29 @@ export class LlrtFunction extends NodejsFunction {
       },
     });
 
+    if (props.useLambdaLayer && !props.llrtBinaryPath) {
```

Comment:
Let's throw a validation error when props.useLambdaLayer is true and props.llrtBinaryPath is specified.